packages/components/src/styles/hidden-links/hidden-links.stories.ts

```diff
@@ -10,14 +10,16 @@ const { generateTemplate } = storybookTemplate('sd-hidden-links');
 const html = String.raw;
 
 /**
- * Can be used to show links only for keyboard users.
+ * Used to show links only for keyboard users.
  *
- * - Per default the hidden links are optimized for a single <sd-button>.
+ * **Related components**:
+ * - [Navigation Item](?path=/docs/components-navigation-item--docs)
```

Comment:
In figma we also have a template called "Hidden Links"

Reply:
In a call the three of us concluded that no need for a template for hidden links